server/server.js

```diff
@@ -0,0 +1 @@
+// testing upstream
```

Comment:
```suggestion
const express = require('express');
const path = require('path');

const app = express();

app.get("/", (req, res) => {
res.end();
});


app.post("/", (req, res) => {
res.end();
});

module.exports = {
app,
}
```
Add some code. Server needs server.

Reply:
hey!